apps/web/src/components/milestone-logs/recruitment/Record.tsx

```diff
@@ -18,80 +18,62 @@ import { AddRecordModal } from '../../display/AddRecordModal';
 import { updateMilestone } from '@services';
 import { emitter, IEN_EVENTS } from '../../../services/event-emitter';
 
+dayjs.extend(duration);
+dayjs.extend(relativeTime);
+
 interface RecordProps {
   job: ApplicantJobRO;
   update: (record?: ApplicantJobRO) => void;
 }
 
+const COMPLETED_STATUSES = [305, 306, 307, 308];
```

Comment:
Will hard coded statuses cause issues here?

Reply:
> Will hard coded statuses cause issues here?

I don't think so, actually it's just refactored not added newly. btw, are you suggesting we should move them to database?

Reply:
I'll add an enum for the codes.

Reply:
we get those 4 values from the master-list.ts, I had added a to-do to fix the hardcoded values